pcsx2/gui/AppMain.cpp

```diff
@@ -294,6 +294,20 @@ void Pcsx2App::PadKeyDispatch( const keyEvent& ev )
 
 	m_kevt.m_keyCode = vkey? vkey : ev.key;
 
+	if (DevConWriterEnabled && m_kevt.GetEventType() == wxEVT_KEY_DOWN) {
+		wxString strFromCode = wxAcceleratorEntry(0, m_kevt.m_keyCode).ToString();
+		if (strFromCode == "\\")
+			strFromCode = "\\\\"; // If copied into PCSX2_keys.ini, \ needs escaping
+
+		// for some reason it doesn't work with extra %ld, so extend the fmt string with it.
+		Console.WriteLn(wxString(L"> Key: %ls%ls%ls%s") + pxsFmt(L" (code: %ld)", m_kevt.m_keyCode),
+			(m_kevt.m_altDown ? L"Alt-" : L""),
+			(m_kevt.m_controlDown ? L"Ctrl-" : L""),
+			(m_kevt.m_shiftDown ? L"Shift-" : L""),
+			strFromCode.c_str()
```

Comment:
Needs to be WX_STR(strFromCode), otherwise it outputs garbage on Windows, and fails to compile on Linux. That's probably why the extra %ld didn't work properly.